torch_geometric/nn/glob/equilibrium_aggregation.py

```diff
@@ -0,0 +1,72 @@
+from typing import List
+
+import torch
+
+EPS = 1e-15
+
+
+class ResNetPotential(torch.nn.Module):
+    def __init__(self, input_size: int, layers: List[int]):
+
+        super().__init__()
+        output_size = 1
+        sizes = [input_size] + layers + [output_size]
+        self.layers = torch.nn.ModuleList([
+            torch.nn.Sequential(torch.nn.Linear(in_size, out_size),
+                                torch.nn.LayerNorm(out_size), torch.nn.Tanh())
+            for in_size, out_size in zip(sizes[:-1], sizes[1:])
+        ])
+
+        self.res_trans = torch.nn.ModuleList([
+            torch.nn.Linear(input_size, layer_size)
+            for layer_size in layers + [output_size]
+        ])
+
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        h = x
+        for layer, res in zip(self.layers, self.res_trans):
+            h = layer(h)
+            h = res(x) + h
+
+        return h
+
+
+class EquilibriumAggregation(torch.nn.Module):
+    """
+    Args:
+        potential (torch.nn.Module): trainable potenial function
+    """
+    def __init__(self, input_dim: int, output_dim: int, layers: List[int],
```

Comment:
```suggestion
    def __init__(self, in_channels: int, out_channels: int, num_layers: List[int],
```
Keep naming consistent :)